src/test/run-make/compiler-lookup-paths/Makefile

```diff
@@ -21,7 +21,7 @@ all: $(TMPDIR)/libnative.a
 	mkdir -p $(TMPDIR)/e1
 	mkdir -p $(TMPDIR)/e2
 	$(RUSTC) e.rs -o $(TMPDIR)/e1/libe.rlib
-	$(RUSTC) e.rs -o $(TMPDIR)/e2/libe.rlib
+	$(RUSTC) e.rs -o $(TMPDIR)/e2/libe-alt.rlib
```

Comment:
Changed because two libraries with the same name and content but different paths will not conflict anymore.